config/projects/datadog-agent.rb

```diff
@@ -83,8 +83,10 @@
   # SysVInit service file
   if redhat?
     extra_package_file '/etc/rc.d/init.d/datadog-agent'
+    runtime_dependency 'libcap2'
   else
     extra_package_file '/etc/init.d/datadog-agent'
+    runtime_dependency 'libcap2-bin'
```

Comment:
Got it, thanks for the explanation. I agree that we shouldn't have `libcap` as a required dependency though.